app/main.py

```diff
@@ -1,14 +1,66 @@
-class Validator:
-    pass
+from abc import abstractmethod, ABC
 
 
-class Number:
-    pass
+class Validator(ABC):
+    def __set_name__(self, owner: object, name: str) -> None:
+        self.protected_name = "_" + name
 
+    def __get__(self, instance: "BurgerRecipe", owner: object) -> int | str:
```

Comment:
Same here

Reply:
I guess no. We can change it like that: obj: Any = None. But third argument must be by syntax anyway. Am I right?